deploy/charts/external-secrets/templates/deployment.yaml

```diff
@@ -51,6 +51,10 @@ spec:
           {{- if .Values.scopedNamespace }}
           - --namespace={{ .Values.scopedNamespace }}
           {{- end }}
+          {{- if and .Values.scopedNamespace .Values.scopedRBAC }}
```

Comment:
I would let this check separate. I can see scenarios where we have one single ESO deployment, but we do not want to allow ClusterSecretStores to be used. Also remember to include `.Values.scopedRBAC` to line [46](https://github.com/external-secrets/external-secrets/pull/800/files#diff-4ad3ee90b9bfebc08396e9117e89863916b79208a9436a8257e67c6d2d653c1eR46) =)

Reply:
updated.